### DISCUSSION TITLE
Any interest in having a guide for how to setup a dev environment for fermentrack?

### DISCUSSION THREAD

**Discussion by mp-se** [thorrak/fermentrack#574] [Ideas]
I wondering if there is any interest to have a guide that describes how to setup a virtual machine that allows for developing and testing fermentrack ?

**Comment by lbussy** [comment]
Always room for more good guides.  I use VMs for my testing, but BPR users would be able to benefit from the same setup.

**Comment by thorrak** [comment]
Definitely. I can write something on the docker vs. non-docker testing setups as well.

**Comment by mp-se** [comment]
I have most of the steps defined so I only need to fill it out with some more words. Im using Ubuntu + vscode as base for my development setup. I will put into the standard fermentrack documentation as a new section.

**Comment by mp-se** [comment]
I've submitted a first version of how I setup my development environment. If you have suggestions we can discuss them here and I will continue to improve this part.
